iothub_service_client/tests/iothub_msging_ll_ut/iothub_messaging_ll_ut.c

```diff
@@ -16,7 +16,28 @@
 #include "umock_c/umocktypes_stdint.h"
 #include "umock_c/umocktypes_bool.h"
 
+static void* my_gballoc_malloc(size_t size)
```

Comment:
maybe not a comment for this PR, but these functions seem to be repeated in a lot of UTs - should they be moved somewhere shareable?

Reply:
They should.
But since this is officially deprecated code, there is no priority to make these change.